### ISSUE TITLE
Unable to find intents.

### ISSUE THREAD

**Issue by ClumsySmurf** [aviadmini/quickimagepick#2]
Running a device (fresh os 5 install) and using multiple sources the intent chooser is empty. I was able to fix this by adding a base intent. 

```
// create chooser intent
        Intent baseIntent = resultIntents.remove(0);
        final Intent result = Intent.createChooser(baseIntent, pTitle);
        result.putExtra(Intent.EXTRA_INITIAL_INTENTS, resultIntents.toArray(new Parcelable[resultIntents.size()]));
```

**Comment by aviadmini** [comment]
Thanks. Probably makes sense to create chooser from an non-empty intent. Will change in next release

**Comment by aviadmini** [comment]
Closed with cb3865b, another chooser fix in d5ecccf
